package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Read-Only-API",
+  "version": "1.0.0",
+  "description": "Bangazon-corp's employees, products and users database",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Should we go ahead and add ```mocha --recursive --watch -R spec```?

Reply:
It will change for mac and pc users a little bit, but I will go ahead and add it